vllm/model_executor/models/glm4_vision_encoder.py

```diff
@@ -79,6 +80,31 @@ def __init__(
         self.output_dropout = torch.nn.Dropout(config.dropout_prob)
 
     def forward(self, x: torch.Tensor) -> torch.Tensor:
+        _ON_NAVI3 = "gfx11" in torch.cuda.get_device_properties("cuda").gcnArchName
```

Comment:
Calling torch.cuda directly in vllm will not work on platforms other than ROCm. Please refer to platform/rocm.py and its superclass. Also is_navi() in utils.py

Reply:
The CK FA implementation is specify to navi3 only.  but is_navi() include all navi GPUs. how about : 
`_ON_NAVI3  = current_platform.is_rocm()  and "gfx11" in torch.cuda.get_device_properties("cuda").gcnArchName `

Reply:
First checking for is_rocm is the right way to go, just please move this alongside the general is_navi, in utils.py for visibility and others to use as well

Reply:
Thanks for the suggestion. I have added is_navi3 function in utils.py